plugins/ufm_log_analyzer_plugin/src/loganalyze/log_analyzers/ibdiagnet2_port_counters_analyzer.py

```diff
@@ -129,7 +129,8 @@ def analyze_iteration_time(self, threshold=0.15):
         filtered_data = filtered_data[["type", "timestamp", "data"]]
         filtered_data["data"] = pd.to_numeric(filtered_data["data"], errors="coerce")
 
-        filtered_data = filtered_data[filtered_data["data"] >= threshold]
+        if self.telemetry_type == "primary":
```

Comment:
Please remove the Redmine link from the PR description and write the why.
All data should be clear when going over the PR description without sending the reader to another source

Reply:
removed